### ISSUE TITLE
Compatibility issue with Loop Grid and Crocoblock JetSmartFilters

### ISSUE THREAD

**Issue by awebguy99** [elementor/elementor#28320]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

This is more to try to reach out to the Elementor devs. Currently, your Loop Grid Infinite Scroll feature is not compatible with Crocoblock's JetSmartFilter.

I've already reached out to Crocoblock and they're saying it's some kind of limitation with Loop Grid. Here's what they're saying;
"_Currently, it is not possible to integrate JetSmartFilters with the pagination of Loop Grid. Our devs would love to add compatibility with the pagination of Loop Grid, but the build of the Loop Grid widget doesn't allow that. Therefore, we cannot expect this feature to arrive soon._"

**Can your devs please reach out to Crocoblock's dev to work out whatever isn't working?**

**To summarise the issue;**
JetSmartFilters is an excellent filtering system, for Woocommerce products etc. But there's some kind of incompatibility / bug with the Loop Grid Infinite Scroll feature. So, right now I have to use the Load More button (not as good UX as infinite scroll) or stop using Loop Grid.

### Steps to reproduce

1. Use Loop Grid with Inifinite Scroll enabled.
2. Setup/use Crocoblock JetSmartFilters.
3. Use a filter to filter products (or I guess any item).
4. Scroll down to the point it would have done an infinite scroll but it doesn't work / no new products displayed.

### Expected behavior

Should keep displaying products/items as you scroll down.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
NA
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by portrayaloflife** [comment]
Just flagging that the fix for this is using the Jet Smart Filters pagination widget instead and placing that below the loop grid and assigning it to it.

**Comment by awebguy99** [comment]
> Just flagging that the fix for this is using the Jet Smart Filters pagination widget instead and placing that below the loop grid and assigning it to it.

But I don't think that's a fix. Unless I'm wrong, the Jet Smart Filters pagination widget does not have an Infinite Scroll feature.

**Comment by awebguy99** [comment]
Any chance the web design community can please get behind this issue?

It's been confirmed with the Crocoblock developers. They're saying they can't make their Jet Smart Filters compatible with Elementor's Loop Grid Infinite Scroll because of an issue with  Elementor's Loop Grid coding.

I'm asking if Elementor's developers can communicate with Crocoblock developers to work out the issue.

**Comment by awebguy99** [comment]
Any chance anyone from Elementor can comment on this? It's a confirmed bug/issue with the Loop Grid, as reported by Crocoblock developers.

**Comment by Pelellone** [comment]
@awebguy99 any solutions about the problem?

**Comment by awebguy99** [comment]
No. It doesn't seem Elementor are interested in fixing this issue or a number of other issues. I'm moving my digital agency to another page builder.

**Comment by StenderIternum** [comment]
Still unable to use elementor loops grid pagination with jetsmartfilters - Either the load more button dublicates or the results are wrong.

**Comment by webcoreinc** [comment]
You def won't get any help here. The Elementor gods could care less.

**Comment by AurelienWebnation** [comment]
Hey!

Try to switch from "Products" Or "Posts" to "Current query" into the loop grid Query settings tab.

<img width="454" height="143" alt="Image" src="https://github.com/user-attachments/assets/93f8c95a-af08-4e00-807f-78f7b89c1e2f" />
